packages/launchpad/cypress/e2e/global-mode-onboarding.cy.ts

```diff
@@ -0,0 +1,44 @@
+describe('Global Mode: Onboarding', () => {
+  beforeEach(() => {
+    cy.scaffoldProject('pristine')
+    cy.openGlobalMode()
+    cy.addProject('pristine')
+    cy.visitLaunchpad()
+  })
+
+  it('scaffolds a project, go back to global screen, and state is cleared', () => {
+    cy.get('[data-cy=project-card]').click()
+    cy.get('[data-cy-testingtype="e2e"]').click()
+    cy.get('h1').should('contain', 'Configuration Files')
+    cy.get('[data-cy="global-mode-link"]').click()
```

Comment:
```suggestion
    cy.findByRole('link', { name: 'Project' }).click()
```
Using the above should remove the need to add data-cy on this anchor tag

Reply:
how does `{ name: 'Project' }` correspond to the `global-mode-link` data-cy attribute?

Reply:
oh you're saying it corresponds to the link element with role attribute `<a role='Project' >...</a>`